rules/windows/privilege_escalation_uac_bypass_diskcleanup_hijack.toml

```
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies User Account Control (UAC) bypass via hijacking DiskCleanup Scheduled Task. Attackers bypass UAC to
+stealthily execute code with elevated permissions.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Bypass UAC via DiskCleanup Scheduled Task Hijack"
+risk_score = 47
+rule_id = "1dcc51f6-ba26-49e7-9ef4-2655abb2361e"
+severity = "medium"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and not event.type:end and
```

```suggestion
event.category:process and event.type:(start or process_started) and
```

for process events, I would be explicit be using "start" because there is also an event.type of info which may result in unrelated or duplicative alerts. process_started is for backwards compat for pre 7.8 beats